ethcore/src/json_tests/state.rs

```diff
@@ -191,3 +200,66 @@ mod state_tests {
 	// https://github.com/ethereum/tests/blob/726b161ba8a739691006cc1ba080672bb50a9d49/GeneralStateTests/stZeroKnowledge2/ecmul_0-3_5616_28000_96.json#L126
 	declare_test!{GeneralStateTest_stZeroKnowledge2, "GeneralStateTests/stZeroKnowledge2/"}
 }
+
+#[cfg(test)]
+mod legacy_state_tests {
```

Comment:
I think we should document why the below tests are considered legacy and why we think they are still worth keeping&running.